src/Makefile.test.include

```diff
@@ -170,6 +170,8 @@ endif
 
 
 BITCOIN_TEST_SUITE += \
+  wallet/test/util.cpp \
+  wallet/test/util.h \
```

Comment:
Why not reuse `wallet/test/wallet_test_fixture.*`?

It already contains `WalletTestingSetup` that we use anyway.